paimon-web-api/src/main/java/org/apache/paimon/web/api/action/context/ActionConf.java

```diff
@@ -0,0 +1,36 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.paimon.web.api.action.context;
+
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+
+/** ActionConf. */
+@Retention(RetentionPolicy.RUNTIME)
+@Target(ElementType.FIELD)
+public @interface ActionConf {
+
+    String value();
+
+    boolean nullable() default true;
+
+    boolean confList() default false;
```

Comment:
Remove this unnecessary interface.

Reply:
They are utilized for null value evaluations and the construction of arguments.
<img width="645" alt="image" src="https://github.com/apache/paimon-webui/assets/35210666/1650216d-de60-45a8-9655-7a5364cc1f1d">
<img width="725" alt="image" src="https://github.com/apache/paimon-webui/assets/35210666/025ba4bf-eaeb-445c-8543-0e54a0739818">